plugins/sonarqube/api-report.md

```diff
@@ -21,14 +21,22 @@ export const EntitySonarQubeCard: (props: {
   duplicationRatings?: DuplicationRating[] | undefined;
 }) => JSX.Element;
 
+// @public (undocumented)
+export const EntitySonarQubeContentPage: (
+  props: SonarQubeContentPageProps,
+) => JSX.Element;
+
 // @public (undocumented)
 export const isSonarQubeAvailable: (entity: Entity) => boolean;
 
 // @public (undocumented)
-export const SonarQubeCard: (props: {
```

Comment:
This was accidentally removed from the exports, right?

Reply:
My bad. Yes it was :). Reverting...